### PR TITLE
[xtask test] Implement `--nocapture` flag.

### PR DESCRIPTION
**Pull request by jimblandy**

### CODE REVIEW

File: xtask/src/test.rs [hunk 1]
```diff
@@ -81,8 +88,8 @@ pub fn run_tests(shell: Shell, mut args: Arguments) -> anyhow::Result<()> {
             "--retries",
             &retries,
         ])
+        .args(nocapture_flags)
         .args(args.finish())
-        .quiet()
```
Comment (ErichDonGubler) [inline xtask/src/test.rs:85]: question: Why are we removing this flag?

**Review by ErichDonGubler — APPROVED** [review]
Tentatively approving, pending a question.

**Comment by cwfitzgerald** [comment]
Why do we need this, it should just pass through excess flags

File: xtask/src/test.rs (hunk 1 above)
Comment (jimblandy) [reply xtask/src/test.rs:85]: Carelessness. I'm sorry.

**Comment by jimblandy** [comment]
I'm sorry, I think this PR is unneeded and a waste of your time.

I think I must have passed `cargo xtask test -- --nocapture`, as is needed for `cargo test`, gotten the complaint "error: failed to parse test binary arguments `--nocapture`: arguments are unsupported", jumped to conclusions, dashed off to fix things, and inadvertently changed my invocation to `cargo xtask test --nocapture`, which works with or without the PR.

**Comment by jimblandy** [comment]
I understand why we need `cargo xtask test`, but when I talk about how inattentive or naive users may be confused by custom harnesses --- I mean *me*

**Pull request closed without merging**